Functions/xASL_bids_BIDS2Legacy.m

```diff
@@ -306,11 +306,22 @@
 
 %% 11. Copy participants.tsv
 
-% Check if participants.tsv exists
-if xASL_exist(fullfile(pathStudy, 'participants.tsv'),'file')==2
-    xASL_Copy(fullfile(pathStudy, 'participants.tsv'),fullfile(pathLegacy, 'Participants.tsv'));
-elseif xASL_exist(fullfile(pathStudy, 'Participants.tsv'),'file')==2
-    xASL_Copy(fullfile(pathStudy, 'Participants.tsv'),fullfile(pathLegacy, 'Participants.tsv'));
+% Define file names
+pathParticipantsTSV = fullfile(pathStudy, 'participants.tsv');
+pathParticipantsTSVxASL = fullfile(pathLegacy, 'participants.tsv');
+
+% Backwards compatibility: rename to lowercase
+fileListParticipantsTSVold = xASL_adm_GetFileList(pathStudy,'^Participants.tsv$',false);
+if ~isempty(fileListParticipantsTSVold)
+    pathParticipantsTSVold = fullfile(pathStudy, 'Participants.tsv');
+    pathParticipantsTSVoldTemp = fullfile(pathStudy, 'Participants_temp.tsv');
+    xASL_Move(pathParticipantsTSVold,pathParticipantsTSVoldTemp);
```

Comment:
Sorry, but I don't understand why this middle file is needed when you do two consecutive moves. Do I miss something?

Reply:
I actually explain this quite detailed here d47a947 ...

> The previous code did not work because windows will complain if you move Participants.tsv to participants.tsv, since it regards both as the same file. The easiest fix is to rename it to another filename temporarily in between. Additionally we had the problem that you can not find the lowercase/uppercase differences using exist or xASL_exist, but xASL_adm_GetFileList uses regular expressions and worked nicely, so I implemented a fix with that. Makes the code a bit clunky though.

... and here ...

https://github.com/ExploreASL/ExploreASL/pull/749#issuecomment-889699476

Reply:
Sorry. Didn't check the commit itself. Makes perfect sense of course!! Can you please add a comment in the code that explains this. Or some guy like me will fix it in the future :D